main_logic/tts_client.py

```
@@ -974,6 +977,17 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
     Returns:
         对应的 TTS worker 函数
     """
+    #指定本地模型的判断方式
+    try:
+        cm = get_config_manager()
+        tts_config = cm.get_model_api_config('tts_custom')
+        user_url = tts_config.get('user_url')
+        # 只要base_url 这里填写了参数 那就确定使用本地服务
+        if user_url and ('http' in user_url or 'ws' in user_url):
+            return local_cosyvoice_worker
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)